webapp/src/ts/services/contact-summary.service.ts

```diff
@@ -69,21 +70,33 @@ export class ContactSummaryService {
   }
 
   get(contact, reports, lineage, targetDoc?) {
-    return this.ngZone.runOutsideAngular(() => this._get(contact, reports, lineage, targetDoc));
+    return this.ngZone.runOutsideAngular(() => {
+      return this
+        ._get(contact, reports, lineage, targetDoc)
+        .catch(error => {
```

Comment:
If we let the exception bubble up, then the contact won't load blocking the user.
But if we catch it here then the summary won't show, an error is printed in console and the contact loads and user can work, perhaps a better user experience.

Reply:
I think it's fine to block loading the contact when contact summary fails.

Reply:
Done